androidClient/app/src/main/java/com/macbitsgoa/comrades/CourseVh.java

```diff
@@ -25,6 +28,7 @@ public void populate(final Course course) {
         nameTv.setText(course.name);
         rootView.setOnClickListener(view -> {
             final Intent courseIntent = new Intent(view.getContext(), CourseActivity.class);
+            courseIntent.putExtra(EXTRA_COURSE_ID, course.id);
```

Comment:
a public static method (in CourseActivity) which takes context and required params and launches CourseActivity would be much handy.

Reply:
what's the problem with this? Intents are standard way of passing data and starting activities.

Reply:
Yes, but it's easy to miss out passing the args. Not a problem now, but can cause little bugs later if someone's not careful.